sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.controller.service.ts

```diff
@@ -57,6 +60,29 @@ export class ScholasticStandingControllerService {
     };
   }
 
+  /**
+   * Get Scholastic Standing summary details.
+   * @param studentId student id to retrieve the scholastic standing summary.
+   * @returns Scholastic Standing summary details.
+   */
+  async getScholasticStandingSummary(options?: {
+    studentId: number;
```

Comment:
Is this optional expecting some future implementation?
Are the methods consuming it not expecting them as mandatory?